### PR TITLE
Getting model generated unit tests to work

### PR DESCRIPTION
**Pull request by spivachuk**

### CODE REVIEW

File: testutil/network/util.go [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+	"github.com/tendermint/tendermint/types"
+	tmtime "github.com/tendermint/tendermint/types/time"
+
+	"github.com/cosmos/cosmos-sdk/server/api"
+	servergrpc "github.com/cosmos/cosmos-sdk/server/grpc"
+	srvtypes "github.com/cosmos/cosmos-sdk/server/types"
+	dclauthtypes "github.com/zigbee-alliance/distributed-compliance-ledger/x/dclauth/types"
+	"github.com/zigbee-alliance/distributed-compliance-ledger/x/dclgenutil"
+)
+
```
Comment (ashcherbakov) [inline testutil/network/util.go:23]: Is it copied from Cosmos (with minor changes)? If so, it makes sense to add a comment that this code is copied from XXX.

File: x/model/client/cli/query_model_test.go [hunk 2]
```diff
@@ -53,47 +51,43 @@ func TestShowModel(t *testing.T) {
 		idVid int32
 		idPid int32
 
-		args []string
-		err  error
-		obj  types.Model
+		common []string
```
Comment (ashcherbakov) [inline x/model/client/cli/query_model_test.go:54]: Can we add negative tests for missing or invalid parameters (flags)?

**Review by ashcherbakov — APPROVED** [review]

File: testutil/network/util.go (hunk 1 above)
Comment (spivachuk) [reply testutil/network/util.go:23]: Added the comments.

File: x/model/client/cli/query_model_test.go (hunk 2 above)
Comment (spivachuk) [reply x/model/client/cli/query_model_test.go:54]: Added TODO comments about necessity to add such negative tests.

**Review by ashcherbakov — APPROVED** [review]

**Pull request merged**